### DISCUSSION TITLE
[DOC] what's the difference between react-strict-dom and expo html elements?

### DISCUSSION THREAD

**Discussion by gasolin** [facebook/react-strict-dom#212] [General]
### Describe the feature request

for a quick glance both projects seem to fill the gap between web & RN development hassles
wonder what's the difference between them

https://github.com/expo/expo/tree/main/packages/html-elements

**Comment by necolas** [comment]
They're very different. `html-elements` is a renaming of React Native components, and still uses React Native for Web. It doesn't do feature polyfilling and can't be used to port web code to native.
